### ISSUE TITLE
"No coverage for file" for projects with coverage

### ISSUE THREAD

**Issue by markis** [ryanluker/vscode-coverage-gutters#75]
I can use extension with the example project in this repo.  But I cannot with my typescript projects that I use istanbul-remap.  Here is an example project:

https://github.com/markis/strip-whitespace

Just clone that repo and run `npm test` then see that coverage gutters reports "No coverage for file" on my source files.

**Comment by markis** [comment]
Nevermind, it was related to windows bash.  Windows bash outputs `/mnt/c` and vscode needs relative paths or `C:\`

**Comment by ryanluker** [comment]
hey @markis relative path resolution for the source files within the lcov is on by default in version 1.x of the extension so you shouldnt have had this issue... I did run the npm test on the project you specified by the info.lcov showed only the test file as having coverage?
![white-strip](https://user-images.githubusercontent.com/1335972/28241825-0c7833e0-6951-11e7-9a4a-65c7ba60bb20.PNG)
and if I did use the display command on the test.js file it did show the lcov coverage correctly
![white-string-1](https://user-images.githubusercontent.com/1335972/28241839-62956fea-6951-11e7-8197-5f3bbf0e3240.PNG)
You probably need to adjust istanbul to also provide coverage for the other files in that project. Hopefully this helps you.
